daemon/pod.go

```diff
@@ -42,11 +43,201 @@ type Pod struct {
 	sync.RWMutex
 }
 
-func NewPod(rawSpec []byte, id string, data interface{}) (*Pod, error) {
+// TODO: remove convertToRunvPodSpec after pod.UserPod is deleted from runv
+func convertToRunvPodSpec(podSpec *apitypes.UserPod) (*pod.UserPod, error) {
```

Comment:
apitypes.UserPod is pod config for network transport, it is user API/ABI. but it is not seen by user directly.
pod.UserPod is pod config which can be created manually by user for composing a pod.

I don't think apitypes.UserPod can replace pod.UserPod. and "UserPod" in apitypes.UserPod is confusing. apitypes.Pod is much better, like all other packagename.Pod.